Translation.Rmd

```diff
@@ -412,7 +416,7 @@ Before we begin, let's quickly cover how formulas are expressed in LaTeX. The fu
   write a fraction you'd use `\frac{a}{b}`. To write a square root, you'd use
   `\sqrt{a}`.
 
-* To group elements together use `{}`: i.e., `x ^ a + b` vs. `x ^ {a + b}`.
+* To group elements together use `{}`: i.e., `x ^ a + b` vsersus `x ^ {a + b}`.
```

Comment:
```suggestion
* To group elements together use `{}`: i.e., `x ^ a + b` versus `x ^ {a + b}`.
```

Reply:
Is so typical of me to add a typo when editing :rofl: